lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/stringListPanel.py

```diff
@@ -19,21 +21,26 @@
     from PySide2.QtGui import QPainter  # type: ignore
     from PySide2.QtWidgets import QCheckBox, QHBoxLayout, QVBoxLayout, QStyle, QStyleOptionHeaderV2, QStylePainter, QWidget  # type: ignore
 
+# TODO: support I8N
+kIncludeAllLabel = "Include all"
+kIncludeAllTooltip = textwrap.dedent(
+    """
+    When enabled, all prims are illuminated. When disabled,
+    only the prims in the Include list are illuminated.
+    """).strip()
```

Comment:
... a bit of overkill to introduce textwrap - just for one line break - imho ?

Reply:
It simplify writing the code by not having to worry about code layout, indentation or it getting messed up by auto code layout.